src/patternfly/demos/Card/templates/cardview-demo--status-card.hbs

```diff
@@ -0,0 +1,70 @@
+{{#> card}}
+  {{#> card-header}}
+    {{#> title titleType="h2" title--modifier="pf-m-lg"}}
+      Status
+    {{/title}}
+  {{/card-header}}
+  {{#> card-body}}
+    {{#> grid grid--modifier="pf-m-all-6-col-on-sm pf-m-all-3-col-on-lg pf-m-gutter"}}
+      {{#> grid-item}}
+        {{#> l-flex l-flex--modifier="pf-m-space-items-sm"}}
+          {{#> l-flex}}
```

Comment:
Did you mean to nest 2 flex layouts here? same for the next grid item after this one. The last 2 aren't nested.

Reply:
Really nice catch